### PR TITLE
Fix chrono deprecation #69

### LINKED ISSUE #69: Fix chrono deprecation warning
`use of deprecated associated function 'chrono::NaiveTime::from_hms_milli': use 'from_hms_milli_opt()' instead`

### PR DESCRIPTION
**Pull request by JasminFragnaud**
Fix #69

### CODE REVIEW

File: src/sentences/bwc.rs [hunk 1]
```diff
@@ -149,7 +149,7 @@ mod tests {
 
         assert_eq!(
             BwcData {
-                fix_time: Some(NaiveTime::from_hms(8, 18, 37)),
+                fix_time: NaiveTime::from_hms_opt(8, 18, 37),
```
Comment (elpiel) [inline src/sentences/bwc.rs:152]: I would suggest to do an `.expect()` on the returned `Option` and wrap it in `Some`. This way if the expected time is wrong in the test itself we will now.

Same goes for all other instances.

**Review by elpiel — CHANGES_REQUESTED** [review]
Thank you very much for the PR!

To make it more robust for bugs in the tests I suggest to you to use `expect` or `unwrap` and wrap the results in `Some`.

File: src/sentences/bwc.rs (hunk 1 above)
Comment (JasminFragnaud) [reply src/sentences/bwc.rs:152]: Something like this ?
`fix_time: Some(NaiveTime::from_hms_opt(8, 18, 37).expect("invalid time"))`

Comment (elpiel) [reply src/sentences/bwc.rs:152]: Yes, exactly!

**Review by elpiel — APPROVED** [review]

**Comment by elpiel** [comment]
Thank you :heart: once again and I hope to see you contributing again in the future!

**Pull request merged**